README.md

```diff
@@ -38,10 +38,21 @@ The idea of these resources is, to offer similar functionality as the
 
 
 ## Build and Execute
+
+Directory: X/riot-saul-coap/RIOT/examples/gcoap/
```

Comment:
Could you add a small statement *when* this directory should be used? We only need it for the `GET` requests, but still have to run the `make` command in the project directory. Also you could change `X/riot-saul-coap` to `.` (which would mean: in this project directory).